src/others/sysmon/lv_sysmon.c

```diff
@@ -181,6 +185,19 @@ static void perf_monitor_event_cb(lv_event_t * e)
            refr_avg_time, render_real_avg_time, flush_avg_time,
            cpu);
 #else
+    if (fps == 0) {
+        info->delay_zero_update_cnt += 1;
+        if (info->delay_zero_update_cnt == 10) {
+            info->delay_zero_update_cnt = 0;
+            info->last_fps = 0;
+        } else {
+            fps = (uint32_t) info->last_fps;
+        }
+    } else {
+        info->delay_zero_update_cnt = 0;
+        info->last_fps = (uint16_t) fps;
+    }
+
```

Comment:
This was to cause the last non zero frame count to drag out for a bit of time so it can be seen instead of flipping to a zero immediately. Again This was only for the purposes of getting an accurate frame count during my test. The tests were really about using the while loop directly after a buffer being passed to the flush function and it defeating the purpose of even having double buffering if that is being done. Any changes I made to the performance monitor portion was only for the purposes of getting an accurate frame count of actual work being done while testing the changed made to the refresh to see if there is any performance gain.